_posts/2016-10-19-katello-puppet4-integration.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: Katello Puppet4 integration
+date: 2016-10-19
+author: Zach Huntington-Meath
+tags:
+- katello
+- puppet 4
```

Comment:
I don't think tags can have a space. I'd just tag it as "puppet" since there's a bunch of other posts w/ that tag.